src/prototype_room_my.js

```diff
@@ -200,6 +200,49 @@ Room.prototype.handleScout = function() {
   }
 };
 
+Room.prototype.checkForEnergyTransfer = function() {
+  let carryHelpInterval = config.carryHelpers.ticksUntilHelpCheck;
+  if (Game.time % carryHelpInterval) {
```

Comment:
yep, it must. Game.time%interval check is true each time exept each (config value) ticks